src/applications/mhv-medications/containers/Prescriptions.jsx

```diff
@@ -472,10 +476,21 @@ const Prescriptions = () => {
             data-testid="Title-Notes"
           >
             When you share your medications list with providers, make sure you
-            also tell them about your allergies and reactions to medications. If
-            you print or download this list, we’ll include a list of your
-            allergies.
+            also tell them about your allergies and reactions to medications.{' '}
+            {!showAllergiesContent && (
+              <span>
+                If you print or download this list, we’ll include a list of your
```

Comment:
For navigating with assistive-tech, its usually better to use a React fragment here to render all this text contained inside the <p> tag.

Reply:
yeah good idea! will do